<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssessmentReportPage</class>
 <widget class="QWidget" name="AssessmentReportPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ÊµãËØÑÊä•Âëä</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #F8F8F8;
}

QPushButton#backButton {
    background-color: transparent;
    color: #07C160;
    border: none;
    font-size: 16px;
}

QPushButton#viewHistoryButton {
    background-color: #07C160;
    color: white;
    border: none;
    border-radius: 22px;
    font-size: 16px;
    font-weight: bold;
    padding: 12px;
}

QPushButton#viewHistoryButton:hover {
    background-color: #06AD56;
}

*
{
    font-family: &quot;Microsoft YaHei&quot;;
}
QPushButton
{
background-color: #07c160; 
color: white; 
border: none; 
border-radius: 8px;
font-size: 14px;
font-weight: bold; 
}

QPushButton#backButton
{
	background-color: transparent; 
	border-image: url(:/image/return.png);
}

QLabel
{
	background-color: transparent; 
}

QProgressBar 
{
	background-color: #f0f0f0;
	border-radius: 5px;
	text-align: center;
}

QProgressBar::chunk 
{
	border-radius: 5px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
         stop:0 #4a90e2, 
         stop:1 #57d9a3);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="headerWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>56</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>56</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#headerWidget {
    background-color: white;
    border-bottom: 1px solid #EEEEEE;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="styleSheet">
         <string notr="true">QLabel {
    font-size: 18px;
    font-weight: bold;
    color: #333333;
}</string>
        </property>
        <property name="text">
         <string>ÊµãËØÑÊä•Âëä</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>375</width>
        <height>756</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>16</number>
       </property>
       <property name="leftMargin">
        <number>16</number>
       </property>
       <property name="topMargin">
        <number>16</number>
       </property>
       <property name="rightMargin">
        <number>16</number>
       </property>
       <property name="bottomMargin">
        <number>16</number>
       </property>
       <item>
        <widget class="QWidget" name="scoreCard" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget#scoreCard {
    background-color: white;
    border-radius: 12px;
    padding: 20px;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">QLabel {
    font-size: 14px;
    color: #999999;
}</string>
            </property>
            <property name="text">
             <string>ÊÇ®ÁöÑÂæóÂàÜ</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="scoreLabel">
            <property name="styleSheet">
             <string notr="true">QLabel {
    font-size: 48px;
    font-weight: bold;
    color: #333333;
}</string>
            </property>
            <property name="text">
             <string>0.0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="scoreProgressBar">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>10</height>
             </size>
            </property>
            <property name="value">
             <number>75</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="riskLevelLabel">
            <property name="styleSheet">
             <string notr="true">QLabel {
    font-size: 20px;
    font-weight: bold;
    color: #07C160;
}</string>
            </property>
            <property name="text">
             <string>Áä∂ÊÄÅËâØÂ•Ω</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="suggestionCard" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget#suggestionCard {
    background-color: white;
    border-radius: 12px;
    padding: 16px;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>8</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">QLabel {
    font-size: 16px;
    font-weight: bold;
    color: #333333;
}</string>
            </property>
            <property name="text">
             <string>üí° ‰∏™ÊÄßÂåñÂª∫ËÆÆ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="suggestionText">
            <property name="styleSheet">
             <string notr="true">QLabel {
    font-size: 14px;
    color: #666666;
    line-height: 1.5;
}</string>
            </property>
            <property name="text">
             <string>Âª∫ËÆÆÂÜÖÂÆπÂ∞ÜÂú®ËøôÈáåÊòæÁ§∫...</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="viewHistoryButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>44</height>
          </size>
         </property>
         <property name="text">
          <string>Êü•ÁúãÂéÜÂè≤Ë∂ãÂäø</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="dimensionsWidget" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget#dimensionsWidget {
    background-color: white;
    border-radius: 12px;
    padding: 16px;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">QLabel {
    font-size: 16px;
    font-weight: bold;
    color: #333333;
}</string>
            </property>
            <property name="text">
             <string>üìä Áª¥Â∫¶ÂàÜÊûê</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="dimensionsContent" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="trendsWidget" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget#trendsWidget {
    background-color: white;
    border-radius: 12px;
    padding: 16px;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">QLabel {
    font-size: 16px;
    font-weight: bold;
    color: #333333;
}</string>
            </property>
            <property name="text">
             <string>üìà ÊÉÖÁª™Ë∂ãÂäø</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="trendsContent" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="indicatorsWidget" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget#indicatorsWidget {
    background-color: white;
    border-radius: 12px;
    padding: 16px;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="styleSheet">
             <string notr="true">QLabel {
    font-size: 16px;
    font-weight: bold;
    color: #333333;
}</string>
            </property>
            <property name="text">
             <string>üéØ ÂÖ≥ÈîÆÊåáÊ†á</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="indicatorsContent" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
